spec/features/subscribe_back_spec.rb

```diff
@@ -15,6 +15,13 @@
     then_i_can_only_go_back_to_govuk
   end
 
+  scenario "confirm email" do
+    when_i_start_signup_via_govuk
+    and_i_select_frequency_and_submit
+    and_i_fill_in_my_email
+    then_i_can_see_a_back_url_with_correct_parameters
```

Comment:
Minor: it's a bit strange to read this in a (UI) feature test, since the user can't actually "see a back URL". How about "then_i_can_go_back_to_the_frequency_page" or similar?